src/Files.App/ViewModels/ISidebarViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Files.App.Extensions;
+using Files.App.Filesystem;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Input;
+
+namespace Files.App.ViewModels
+{
+	public interface ISidebarViewModel : INotifyPropertyChanged, IDisposable
```

Comment:
Can you add docs/comments to the interface?

Reply:
Why is sidebar viewmodel an interface?